src/main/java/Command.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Represents as an outline what a command carried about by
+ * Chatbot should be able to do.
+ * Any class that extends this class must implement method
+ * execute and isExit.
+ * Only inherits from this class according to what Chatbot can
+ * understand as a command.
```

Comment:
Great use of Javadocs to explain the purpose of each class!

Reply:
thanks for the comment!